### PR TITLE
Add device model

### PR DESCRIPTION
**Pull request by davin-cn**

### CODE REVIEW

File: dtmi/ip3/hrm-1.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+      "schema": "double",
+      "description": "thr frq of telemetry.",
+      "writable": true
+    },
+    {
+      "@type": [ "Telemetry", "Power" ],
+      "name": "power",
+      "displayName": "TotalWattage",
+      "unit": "kilowatt",
+      "description": "the electricity comsumption,ex.1000kwh",
```
Comment (digimaun) [inline dtmi/ip3/hrm-1.json:34]: Do you want to capitalize 'The` ? also `consumption` is mispelled.

File: dtmi/ip3/hrm-1.json [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+      "name": "sn",
+      "displayName": "Device SN",
+      "description": "Device Serials Number.ex S202104150006",
+      "schema": "string"
+    },
+    {
+      "@type": "Property",
+      "name": "sw",
+      "displayName": "Software Version",
+      "description": "the device software version,ex.sw:NB.1.1.2",
```
Comment (digimaun) [inline dtmi/ip3/hrm-1.json:19]: This and the next property I'm  wondering if the first letter should be capitalized.

File: dtmi/ip3/hrm-1.json [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+      "schema": "string"
+    },
+    {
+      "@type": "Property",
+      "name": "TeleFrq",
+      "schema": "double",
+      "description": "thr frq of telemetry.",
```
Comment (digimaun) [inline dtmi/ip3/hrm-1.json:26]: Should 'thr' be 'The'?

**Review by rido-min — CHANGES_REQUESTED** [review]
@davin-cn can you check the comments in this PR?

File: dtmi/ip3/hrm-1.json (hunk 3 above)
Comment (davin-cn) [reply dtmi/ip3/hrm-1.json:26]: ok ,tks

**Review by davin-cn — COMMENTED** [review]
change the hrm model

File: dtmi/ip3/hrm-1.json [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+      "name": "sw",
+      "displayName": "Software Version",
+      "description": "The device software version,ex.sw:NB.1.1.2.",
+      "schema": "string"
+    },
+    {
+      "@type": "Property",
+      "name": "teleFrq",
+      "schema": "double",
+      "description": "The frq of telemetry.",
```
Comment (digimaun) [inline dtmi/ip3/hrm-1.json:26]: Does it make sense to spell out what `frq` means?

File: dtmi/ip3/hrm-1.json [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+      "@type": "Property",
+      "name": "sw",
+      "displayName": "Software Version",
+      "description": "The device software version,ex.sw:NB.1.1.2.",
+      "schema": "string"
+    },
+    {
+      "@type": "Property",
+      "name": "teleFrq",
+      "schema": "double",
```
Comment (digimaun) [inline dtmi/ip3/hrm-1.json:25]: Are you OK with this property missing a `displayName`?

**Review by digimaun — COMMENTED** [review]
Have a couple suggestions for improvement but looks fine otherwise.

File: dtmi/ip3/hrm-1.json (hunk 5 above)
Comment (davin-cn) [reply dtmi/ip3/hrm-1.json:25]: OK,you can do that.

**Review by davin-cn — COMMENTED** [review]
ok

**Review by rido-min — APPROVED** [review]

**Pull request merged**